prysm/Dockerfile.prysm

```diff
@@ -1,28 +1,24 @@
 FROM golang:1.18 as builder
 
-COPY ./prysm/go.mod ./prysm/go.sum /app/prysm/
+RUN apt-get update
+RUN go install github.com/bazelbuild/bazelisk@latest
 
 WORKDIR /app/prysm
-
-RUN go mod download
-
 COPY ./prysm /app/prysm
 
-WORKDIR /app/prysm
 
 # The flag below may be needed if blst throws SIGILL, which happens with certain (older) CPUs
 # ENV CGO_CFLAGS="-O -D__BLST_PORTABLE__"
 ENV CGO_CFLAGS=$CGO_CFLAGS
 
-RUN go build -o /build/beacon-node -buildvcs=false ./cmd/beacon-chain
-RUN go build -o /build/validator -buildvcs=false ./cmd/validator
-RUN go build -o /build/bootnode -buildvcs=false ./tools/bootnode
+RUN apt-get install patch
+RUN bazelisk build //cmd/beacon-chain //cmd/validator
 
 FROM ubuntu:20.04
 RUN apt-get update && apt-get install -y curl jq iproute2
 
-COPY --from=builder /build/beacon-node /usr/local/bin/
-COPY --from=builder /build/validator /usr/local/bin/
+COPY --from=builder /app/prysm/bazel-out/*-fastbuild/bin/cmd/beacon-chain/beacon-chain_/beacon-chain /usr/local/bin/
```

Comment:
I don't think globbing works with the COPY command.

Reply:
It is supported actually: https://docs.docker.com/engine/reference/builder/#copy